src/sentry/buffer/redis.py

```diff
@@ -225,6 +229,10 @@ def incr(self, model, columns, filters, extra=None):
                 # (this is to ensure a zero downtime deploy where we can transition event processing)
                 pipe.hset(key, "e+" + column, pickle.dumps(value))
                 # pipe.hset(key, 'e+' + column, json.dumps(self._dump_value(value)))
```

Comment:
this might have been broken or something which is why it was disabled - dont actually remember